File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/response/DownloadSettingsResponseContent.java
package de.dimedis.mobileentry.backend.response;

import com.google.gson.annotations.SerializedName;

public class DownloadSettingsResponseContent extends BaseResponseContent {
    @SerializedName("heartbeat_interval_idle")
    public int heartbeatIntervalIdle;
    @SerializedName("heartbeat_interval_on_duty")
    public int heartbeatIntervalOnDuty;
    @SerializedName("offline_detect_timeout")
    public int offlineDetectTimeout;
    @SerializedName("offline_detect_count")
    public int offlineDetectCount;
    @SerializedName("scan_ok_switch_delay")
    public int scanOkSwitchDelay;
    @SerializedName("scan_denied_switch_delay")
    public int scanDeniedSwitchDelay;
    @SerializedName("scan_cancel_timeout")
    public int scanCancelTimeout;
    @SerializedName("scan_doublescan_delay")
    public int scanDoubleScanDelay;

    @Override
    public String toString() {
        return "DownloadSettingsResponseContent{" +
                "heartbeatIntervalIdle=" + heartbeatIntervalIdle +
                ", heartbeatIntervalOnDuty=" + heartbeatIntervalOnDuty +
                ", offlineDetectTimeout=" + offlineDetectTimeout +
                ", offlineDetectCount=" + offlineDetectCount +
                ", scanOkSwitchDelay=" + scanOkSwitchDelay +
                ", scanDeniedSwitchDelay=" + scanDeniedSwitchDelay +
                ", scanCancelTimeout=" + scanCancelTimeout +
                ", scanDoubleScanDelay=" + scanDoubleScanDelay +
                '}';
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/response/DownloadSettingsResponseContent.kt
package de.dimedis.mobileentry.backend.response

import com.google.gson.annotations.SerializedName

open class DownloadSettingsResponseContent : BaseResponseContent() {

    @SerializedName("heartbeat_interval_idle")
    var heartbeatIntervalIdle = 0

    @SerializedName("heartbeat_interval_on_duty")
    var heartbeatIntervalOnDuty = 0

    @SerializedName("offline_detect_timeout")
    var offlineDetectTimeout = 0

    @SerializedName("offline_detect_count")
    var offlineDetectCount = 0

    @SerializedName("scan_ok_switch_delay")
    var scanOkSwitchDelay = 0

    @SerializedName("scan_denied_switch_delay")
    var scanDeniedSwitchDelay = 0

    @SerializedName("scan_cancel_timeout")
    var scanCancelTimeout = 0

    @SerializedName("scan_doublescan_delay")
    var scanDoubleScanDelay = 0

    override fun toString(): String {
        return "DownloadSettingsResponseContent{" +
                "heartbeatIntervalIdle=" + heartbeatIntervalIdle +
                ", heartbeatIntervalOnDuty=" + heartbeatIntervalOnDuty +
                ", offlineDetectTimeout=" + offlineDetectTimeout +
                ", offlineDetectCount=" + offlineDetectCount +
                ", scanOkSwitchDelay=" + scanOkSwitchDelay +
                ", scanDeniedSwitchDelay=" + scanDeniedSwitchDelay +
                ", scanCancelTimeout=" + scanCancelTimeout +
                ", scanDoubleScanDelay=" + scanDoubleScanDelay +
                '}'
    }
}